test/message_test.rb

```diff
@@ -353,6 +353,32 @@ def test_decode_repeated_strings
     assert_equal 0xCAFE, obj.b
   end
 
+  def test_encode_repeated_utf8_strings
+    init = -> (x) do
+      x.a = 1234
+      x.names[0] = "™"
+      x.names[1] = "€"
+      x.names[2] = "£"
+      x.names[3] = "∞"
+      x.b = 0xCAFE
+    end
+
+    expected_data = ::RepeatedStrings.encode(::RepeatedStrings.new.tap(&init))
+    actual_data = RepeatedStrings.encode(RepeatedStrings.new.tap(&init))
+
+    assert_equal expected_data, actual_data
+  end
```

Comment:
Could maybe use a short comment to explain what this test is checking. Same for the following test.

Reply:
Done.